opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -88,9 +89,11 @@ namespace Opm {
 
         const TableManager& getTableManager() const;
 
-        // the unit system used by the deck. note that it is rarely needed to
-        // convert units because internally to opm-parser everything is
-        // represented by SI units...
+        std::vector< int > getRegions( const std::string& kw = "FIPNUM" ) const;
```

Comment:
Please leave out the default keyword - there is already a context aware `defaultRegion()` in the `EclipseState`.